app/Filament/Resources/DatabaseHostResource/Pages/ListDatabaseHosts.php

```diff
@@ -31,14 +31,17 @@ public function table(Table $table): Table
                 TextColumn::make('port')
                     ->sortable(),
                 TextColumn::make('username')
+                    ->icon('tabler-user')
                     ->searchable(),
-                TextColumn::make('max_databases')
-                    ->numeric()
-                    ->sortable(),
+                TextColumn::make('databases_count')
+                    ->counts('databases')
+                    ->icon('tabler-database')
+                    ->label('Databases'),
                 TextColumn::make('node.name')
-                    ->numeric()
+                    ->icon('tabler-server-2')
                     ->sortable(),
```

Comment:
Why should add a placeholder here so the column isn't empty when no node is linked.

Reply:
Good idea!

Reply:
I tried it out and I don't think it looks better.

<img width="1515" alt="image" src="https://github.com/user-attachments/assets/98e1d337-3a23-42c5-82be-d860e43f3b14">

<img width="1503" alt="image" src="https://github.com/user-attachments/assets/b59a8c4e-b5ae-4109-80a5-5b23682b76ce">

Reply:
Did you use `->placeholder`?